install_cardio.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+# encoding: utf-8
+# This script adds card.io to the swift example app.
+
+begin
+  gem "xcodeproj"
+rescue LoadError
+  system("gem install xcodeproj")
+  Gem.clear_paths
+end
+
+require 'xcodeproj'
+
+# puts '▸ Installing card.io – this may take a while'
+# open('Example/Cartfile', 'a') { |f|
+#   f.puts "github \"card-io/card.io-iOS-source\""
+# }
+# system('cd Example; carthage update --platform ios')
+
+project_path = 'Example/Stripe iOS Example (Simple).xcodeproj'
+project = Xcodeproj::Project.open(project_path)
+
+# add framework
+lib_path = 'Carthage/Build/iOS/CardIO.framework'
+libRef = project['Frameworks'].new_file(lib_path);
+framework_buildphase = project.objects.select {
+  |x| x.class == Xcodeproj::Project::Object::PBXFrameworksBuildPhase
+}[0];
+framework_buildphase.add_file_reference(libRef);
```

Comment:
I wonder if we can shorten this script a little bit by defining it as an optional framework:

<img width="686" alt="screen shot 2017-06-08 at 4 29 36 pm" src="https://user-images.githubusercontent.com/28276156/26954891-caf56a88-4c67-11e7-9c32-3bb2f7a5a15d.png">